image-create

```diff
@@ -207,7 +207,14 @@ try:
         memory_mb = 3072
     else:
         memory_mb = 2048
-    machine = testvm.VirtMachine(verbose=args.verbose, image=args.image, memory_mb=memory_mb, maintain=True)
+
+    console_file = tempfile.NamedTemporaryFile(suffix='.log', prefix='console-').name
```

Comment:
Not keeping the `NamedTemporaryFile` object around will immediately close it, and thus also remove the file again. So the only effect of this is that you keep a random name. It's not super critical here, but please don't learn this pattern -- it's actively dangerous in "production" applications. Drop the `.name` here, and pass `console_file.name` to VirtMachine instead, so that the TemporaryFile object survives throughout image-create.